soc/arm/nxp_imx/mcimx8mm_a53/mmu_regions.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <arch/arm/aarch64/arm_mmu.h>
+
+#define SZ_1K	1024
+
+static const struct arm_mmu_region mmu_regions[] = {
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 0),
+			      DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 0),
+			      MT_DEVICE_nGnRnE | MT_RW | MT_NS),
```

Comment:
instead of `MT_RW` please use `MT_P_RW_U_RW` or similar.

Reply:
fix in next version